crates/apollo-compiler/src/database/hir.rs

```diff
@@ -853,6 +874,31 @@ impl Value {
     pub fn is_variable(&self) -> bool {
         matches!(self, Self::Variable(..))
     }
+
+    /// Returns `true` if `other` represents the same value as `self`. This is different from the
+    /// `Eq` implementation as it ignores location information.
+    pub fn is_same_value(&self, other: &Value) -> bool {
```

Comment:
def different than `Eq`, butttt could it be a `PartialEq`? or should we save `PartialEq` for semantic equality?

Reply:
i've forgotten about that, ty!